plugins/scaffolder-backend/src/scaffolder/dryrun/createDryRunner.ts

```diff
@@ -119,6 +119,9 @@ export function createDryRunner(options: TemplateTesterCreateOptions) {
           ],
           templateInfo: {
```

Comment:
I think that we should probably move this `templateInfo` block entirely to the caller, and pass this entire block in rather than just `templateMetadata`.

Does that make sense? :pray:

Reply:
Yeah I think I got it. Thank you for the comment. Will knock that out tonight or tomorrow.

Reply:
So I went ahead and did this but I didn't feel it was worth it (at least the way I implemented it) because I had to extract the `dryrunid` and `contentsPath` from the baseUrl. I know there has to be a better way here but it just didn't come to me in the moment. 

Let me know what you think and I will give it another shot.

Reply:
This definitely made it easier to test.